WooCommerce/WooCommerceTests/ViewRelated/Orders/Simple Payments/SimplePaymentsSummaryViewModelTests.swift

```diff
@@ -89,16 +103,176 @@ final class SimplePaymentsSummaryViewModelTests: XCTestCase {
         // Given
         let currencyFormatter = CurrencyFormatter(currencySettings: CurrencySettings()) // Default is US.
-        let viewModel = SimplePaymentsSummaryViewModel(providedAmount: "100", totalWithTaxes: "104.30", taxAmount: "$4.3", currencyFormatter: currencyFormatter)
+        let viewModel = SimplePaymentsSummaryViewModel(providedAmount: "100",
+                                                       totalWithTaxes: "104.30",
+                                                       taxAmount: "$4.3",
+                                                       taxLines: [],
+                                                       currencyFormatter: currencyFormatter)
 
         // When & Then
         XCTAssertEqual(viewModel.taxRate, "4.30")
     }
 
+    func test_showTaxBreakup_is_false_if_taxLinesInSimplePayments_feature_flag_is_off() {
+        // Given
+        let featureFlagService = MockFeatureFlagService(isTaxLinesInSimplePaymentsOn: false)
+        let viewModel = SimplePaymentsSummaryViewModel(providedAmount: "100",
+                                                       totalWithTaxes: "104.30",
+                                                       taxAmount: "$4.3",
+                                                       taxLines: [],
+                                                       featureFlagService: featureFlagService)
+        // Then
+        XCTAssertFalse(viewModel.showTaxBreakup)
+    }
+
+    func test_showTaxBreakup_is_true_if_taxLinesInSimplePayments_feature_flag_is_on() {
+        // Given
+        let featureFlagService = MockFeatureFlagService(isTaxLinesInSimplePaymentsOn: true)
+        let viewModel = SimplePaymentsSummaryViewModel(providedAmount: "100",
+                                                       totalWithTaxes: "104.30",
+                                                       taxAmount: "$4.3",
+                                                       taxLines: [],
+                                                       featureFlagService: featureFlagService)
+        // Then
+        XCTAssertTrue(viewModel.showTaxBreakup)
+    }
+
+    func test_showChargeTaxesToggle_is_true_if_taxLinesInSimplePayments_feature_flag_is_off() {
+        // Given
+        let featureFlagService = MockFeatureFlagService(isTaxLinesInSimplePaymentsOn: false)
+        let viewModel = SimplePaymentsSummaryViewModel(providedAmount: "100",
+                                                       totalWithTaxes: "104.30",
+                                                       taxAmount: "$4.3",
+                                                       taxLines: [],
+                                                       featureFlagService: featureFlagService)
+        // Then
+        XCTAssertTrue(viewModel.showChargeTaxesToggle)
+    }
+
+    func test_when_taxLines_is_not_empty_and_taxLinesInSimplePayments_feature_flag_is_off_then_showChargeTaxesToggle_is_true() {
```

Comment:
Nit: this can be renamed to put the expected behavior before the conditions to make it clearer. So we can name this to make easier to follow and consistent with the naming of other test cases:
```suggestion
    func test_showChargeTaxesToggle_is_true_when_taxLines_is_not_empty_and_taxLinesInSimplePayments_feature_flag_is_off() {
```

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/5943/commits/50f48dc47d31ae10c83b4727969eb2a89aba3fe3